aspnetcore/razor-pages/ui-class.md

```diff
@@ -319,7 +319,7 @@ The *_ViewStart.cshtml* file is required to use the layout of the Razor Pages pr
   dotnet build RazorUIClassLib
   ```
 
-The build output contains *RazorUIClassLib.dll* and *RazorUIClassLib.Views.dll*. *RazorUIClassLib.Views.dll* contains the compiled Razor content.
+In ASP.NET Core version 6.0 or later, the build output contains `RazorUIClassLib.dll`. In versions of ASP.NET Core earlier than 6.0, the build output contains `RazorUIClassLib.dll` and `RazorUIClassLib.Views.dll`. `RazorUIClassLib.Views.dll` contains the compiled Razor content.
```

Comment:
Can you revert this and I'll make those changes in #22013

In this repo, we never say in version x and later ...
In previous version ....

We use monikers. That was a decision the PU made with V 1.0